toolkit/imageconfigs/scripts/setup_cvm_image.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+EFIDIR="BOOT"
+KERNEL_VERSION=""
+
+# Image generation is done in a chroot environment, so running `uname -r`
+# will return the version of the host running kernel. This function works
+# under the assumption that exactly one kernel is installed in the end image.
+get_kernel_version() {
+    kernel_modules_dir="/usr/lib/modules"
+    KERNEL_VERSION="$(ls $kernel_modules_dir)"
+}
+
+# symlink /boot/efi to ../efi
+cp -a /boot/efi/. /efi
```

Comment:
Who expects the esp partition at /efi?

Reply:
systemd-boot, which is what we're using over grub for CVM images